java/app/grapheneos/setupwizard/action/WelcomeActions.kt

```diff
@@ -89,4 +95,82 @@ object WelcomeActions {
         if (localeInfo != null) adapter.insert(localeInfo, 0)
         return adapter
     }
+
+    private fun disableOemUnlockByUser() {
+        getOemLockManager()?.isOemUnlockAllowedByUser = false
+        refreshOemUnlockStatus()
+    }
+
+    private fun getOemLockManager(): OemLockManager? {
+        return appContext.getSystemService(OemLockManager::class.java)
+    }
+
+    private fun rebootBootloader() {
```

Comment:
rebootToBootloader would be a better name. Bootloader reboot is a separate thing (when device is in bootloader interface, bootloader itself can be rebooted)

Reply:
sounds good